### PR TITLE
Dependency Updates, and fix scripts

### PR DESCRIPTION
**Pull request by tofixx**
- replace tslint by eslint/prettier
- update dependencies
- fix path mappings

### CODE REVIEW

**Review by CeEv — DISMISSED** [review]

**Review by CeEv — APPROVED** [review]

**Review by CeEv — CHANGES_REQUESTED** [review]

File: src/interfaces/Mail.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default interface Mail {
 	subject: string;
 	text: string;
 	html: string;
-	attachments?: Array<{ content: Buffer | string, filename: string }>;
+	attachments?: Array<{ content: Buffer | string; filename: string }>;
```
[the remark this answers is not in the document]
Comment (tofixx) [reply src/interfaces/Mail.ts:8]: As this is an interface, semicolon is valid here (as for the properties around). https://medium.com/swlh/typescript-best-practices-delimiters-and-ordering-7feb42e0ca1d#:~:text=In%20TypeScript%2C%20interfaces%20and%20type,a%20comma%20or%20a%20semicolon.&text=They're%20both%20valid%2C%20but%20TypeScript%20prefers%20semicolon.

File: src/routes/message.ts [hunk 2]
```diff
@@ -52,16 +46,8 @@ router.post('/:messageId/seen', async (req, res) => {
...
-			req.body.receiverId,
-		);
-		if (
-			req.body.redirect &&
-			message &&
-			message.payload &&
-			req.body.redirect in message.payload
-		) {
+		const message = await messageService.seen(req.params.messageId, req.body.receiverId);
+		if (req.body.redirect && message && message.payload && req.body.redirect in message.payload) {
```
[the remark this answers is not in the document]
Comment (tofixx) [reply src/routes/message.ts:50]: We have a line-length defined, which is `printWidth: 120`. This line has a width of 103 only.

File: src/interfaces/Mail.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (CeEv) [reply src/interfaces/Mail.ts:8]: ok

File: src/models/message.ts [hunk 3]
```diff
@@ -20,28 +20,34 @@ export const userResourceSchema = new mongoose.Schema({
...
-	payload: Object,
-	languagePayloads: [
-		{
-			language: String,
-			payload: Object,
+	{ timestamps: { createdAt: true, updatedAt: false } }
+);
+
+export const messageSchema = new mongoose.Schema(
+	{
```
[the remark this answers is not in the document]
Comment (tofixx) [reply src/models/message.ts:31]: sure, but specific things in prettier may be overridden by eslint

**Review by CeEv — APPROVED** [review]

**Pull request merged**